src/ontology/fbbt-edit.obo

```diff
@@ -145911,6 +145911,50 @@ intersection_of: part_of FBbt:00003210 ! adult peripheral fat mass
 property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
 property_value: http://purl.org/dc/terms/date 2022-08-30T13:08:51Z xsd:dateTime
 
+[Term]
+id: FBbt:00059157
+name: cyst cell of female germline 2-cell cyst
+def: "One of the two cyst cells that make up a 2-cell female germline cyst, resulting from the first division of a cystoblast (Hinnant et al., 2020)." [FlyBase:FBrf0244981]
+subset: cur
+intersection_of: FBbt:00004877 ! cyst cell of ovary
+intersection_of: part_of FBbt:00048531 ! female germline 2-cell cyst
+relationship: develops_from FBbt:00004876 ! cystoblast
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-08-30T16:22:58Z xsd:dateTime
+
+[Term]
+id: FBbt:00059158
+name: cyst cell of female germline 4-cell cyst
+def: "One of the four cyst cells that make up a 4-cell female germline cyst, resulting from the second round of cystoblast division (Hinnant et al., 2020)." [FlyBase:FBrf0244981]
+subset: cur
+intersection_of: FBbt:00004877 ! cyst cell of ovary
+intersection_of: part_of FBbt:00048532 ! female germline 4-cell cyst
+relationship: develops_from FBbt:00059157 ! cyst cell of female germline 2-cell cyst
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-08-30T16:25:39Z xsd:dateTime
+
+[Term]
+id: FBbt:00059159
+name: cyst cell of female germline 8-cell cyst
+def: "One of the eight cyst cells that make up a 8-cell female germline cyst, resulting from the third round of cystoblast division (Hinnant et al., 2020)." [FlyBase:FBrf0244981]
+subset: cur
+intersection_of: FBbt:00004877 ! cyst cell of ovary
+intersection_of: part_of FBbt:00048533 ! female germline 8-cell cyst
+relationship: develops_from FBbt:00059158 ! cyst cell of female germline 4-cell cyst
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-08-30T16:26:38Z xsd:dateTime
+
+[Term]
+id: FBbt:00059160
+name: cyst cell of female germline 16-cell cyst
+def: "One of the 16 cyst cells that make up a female germline 16-cell cyst, resulting from the fourth and last round of cystoblast division (Hinnant et al., 2020)." [FlyBase:FBrf0244981]
+subset: cur
+intersection_of: FBbt:00004877 ! cyst cell of ovary
+intersection_of: part_of FBbt:00048534 ! female germline 16-cell cyst
+relationship: develops_from FBbt:00059159 ! cyst cell of female germline 8-cell cyst
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-08-30T16:50:20Z xsd:dateTime
+
```

Comment:
Looks like 'presumptive oocyte' and 'presumptive nurse cell' need to be subclasses of 'cyst cell of female germline 16-cell cyst'.

Reply:
Indeed.

Another related change is that the `female germline 16-cell cyst`, currently said to be `part_of some ovariole`, can be more precisely said to be `part_of some germarium`. By the time the 16-cell cyst leaves the germarium, it has already become a `egg chamber`, so the 16-cell cyst is entirely within the germarium.

Reply:
>By the time the 16-cell cyst leaves the germarium, it has already become a `egg chamber`

Yeah, I have some recollection that I might have changed this at some point, because I think it is referred to as an "egg chamber" AFTER leaving the germarium, so during the transition phase, while it is leaving (and not fully part of the germarium), it would still be a "16-cell cyst". However, we could alternatively consider it to be an extrusion of (and still part of) the germarium until it is an "egg chamber" - so you can change if you like.